src/Core/Program.cs

```diff
@@ -822,5 +825,15 @@ public void AddGlobalField(Address address, DataType dt, string name)
             }
             fields.Add(offset, dt, name);
         }
+
```

Comment:
In PE binaries, external procedures always are associated with a module name. On other platforms, like e.g. ELF, they are not. This signature should probably be changed to accept external procedures with `null` module names.

Reply:
Also, let's add a doc comment here as well.

Reply:
Done in 4c6da22